apps/frontend/src/pages/project/project.tsx

```diff
@@ -119,14 +119,38 @@ const Project = (): JSX.Element => {
 	const [contributorToSplit, setContributorToSplit] =
 		useState<ContributorGetAllItemResponseDto | null>(null);
 
+	const hasViewPermission =
+		checkHasPermission(
+			[PermissionKey.VIEW_ALL_PROJECTS, PermissionKey.MANAGE_ALL_PROJECTS],
+			userPermissions,
+		) ||
+		checkIsProjectPermitted({
```

Comment:
maybe we can pass array or permissions to checkIsProjectPermitted 🤔

Reply:
Yeah. It`s local helper, so I just refactored it to receive array of permissions.